openshift/default-catalog-consistency/test/utils/utils.go

```diff
@@ -0,0 +1,73 @@
+package utils
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+	"strings"
+
+	apiv1 "github.com/operator-framework/operator-controller/api/v1"
+)
+
+// LoadImagesFromCatalogs reads the catalogs from the files used and returns a list of image references.
+func LoadImagesFromCatalogs() ([]string, error) {
+	var images []string
+	catalogsPath := "../../../catalogd/kustomize/overlays/openshift/catalogs"
```

Comment:
it could be worth making the the path a parameter to the function and configurable at the suite level maybe through env var or something like that. Just a thought/suggestion...